### ISSUE TITLE
Session: The landscape of state space models and methods: HMMs, DBNs etc

### ISSUE THREAD

**Issue by paul1010** [ACEMS/ECRretreat2017Nov#10]
State space models are applied broadly across fields including computer science, statistics and engineering, using a wide array of methods for their estimation and inference. Despite the plethora of techniques, there are some important similarities between HMMs, Dynamic Bayesian Networks (DBNs) and even Kalman Filters. See for instance [Murphy, 2002](https://pdfs.semanticscholar.org/60ed/db80f54c796750a8173f2abea3bc85a62322.pdf).

A discussion on state space models could be valuable for identifying areas of overlapping interest, and potential opportunities for research on methods as driven by application to new domains. Such a discussion could include:
- the types of state space models (what is captured, discrete/continuous etc)
- the methods for their estimation (e.g. expectation-maximisation to RJMCMC for non-homogeneous DBNs)
- the methods for inference (e.g. Viterbi for HMM, it's relationship to max-sum inference, extending to max-product for DBN inference, and Bayesian HMMs)
- opportunities for collaboration and new research

**Comment by jesse-jesse** [comment]
Hi Paul, Do you have any problems that exist in this area specifically?

**Comment by paul1010** [comment]
@jesse-jesse Yes - including how to incorporate data and parameter uncertainty for inference in a computationally efficient manner (akin in some ways to Variational Bayes - they share the message passing algorithm for instance)
